app/controllers/openid_connect/certs_controller.rb

```diff
@@ -0,0 +1,9 @@
+module OpenidConnect
+  class CertsController < ApplicationController
+    def index
+      expires_in 1.week, public: true
```

Comment:
I don't think so, it just means if/when we choose to rotate our private/public key, we should wait at least a week before removing the old one?